x-pack/plugins/enterprise_search/public/applications/app_search/components/crawler/crawler_overview_logic.ts

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { kea, MakeLogicType } from 'kea';
+
+import { flashAPIErrors, flashSuccessToast } from '../../../shared/flash_messages';
+
+import { HttpLogic } from '../../../shared/http';
+import { EngineLogic } from '../engine';
+
+import { CrawlerLogic } from './crawler_logic';
+import { CrawlerDataFromServer, CrawlerDomain } from './types';
+import { crawlerDataServerToClient, getDeleteDomainSuccessMessage } from './utils';
+
+interface CrawlerOverviewActions {
+  deleteDomain(domain: CrawlerDomain): { domain: CrawlerDomain };
+}
+
+export const CrawlerOverviewLogic = kea<MakeLogicType<{}, CrawlerOverviewActions>>({
```

Comment:
Seems like this export is unused and the file should be removed?

Reply:
Yep. thanks!